engine/schema/src/main/java/com/cloud/vm/NicVO.java

```diff
@@ -330,7 +333,7 @@ public String toString() {
             .append(reservationId)
             .append("-")
             .append(iPv4Address)
-            .append("]")
+            .append(mtu).append("]")
```

Comment:
is it intentionally appended without a separator?

Reply:
this was added temporarily for some debugging, will remove it.